keps/sig-node/277-ephemeral-containers/README.md

```diff
@@ -629,10 +637,8 @@ via this subresource. `EphemeralContainerStatuses` is updated in the same manner
 as everything else in `Pod.Status` via `/status`.
 
 `Pod.Spec.EphemeralContainers` may be updated via `/ephemeralcontainers` as per
-normal (using PUT, PATCH, etc) except that existing Ephemeral Containers may not
-be modified or deleted. Deleting Ephemeral Containers is not supported in the
-initial implementation to reduce complexity. It could be added in the future,
-but see *Killing Ephemeral Containers* below for additional constraints.
+normal (using PUT, PATCH, etc) except that existing Ephemeral Containers may
+not be modified.
```

Comment:
Since we removed delete the existing ephemeral container above, are you referring that the policy is the deletion / removal of an ephemeral container is permitted?

Reply:
That's correct, we'll permit deleting ephemeral containers.